tests/test_exe.py

```diff
@@ -153,6 +154,15 @@ def test_hash(
             ), f"The {actual_digest.name} digest did not match."
 
 
+def maybe_platform_args() -> Iterator[str]:
+    # N.B.: When running on Windows ARM64 we need to use an x86-64 PBS and so the science
+    # executable thinks its running on x86-64 as a result. We force the platform when building
+    # scies on Windows ARM64 to work around this mis-identification in dogfood-style tests.
```

Comment:
Ok, this is a bug. The science scie needs to ship with the correction for this. Since the scie-jump powering the science scie is not emulated - it is an aarch64 Windows binary - its `{scie.platform}` / `{scie.platform.os}` / `{scie.platform.arch}` placeholders should be used to plumb a corrected setting for the current platform to the science Python code.

Reply:
Ok, fixed.